CONTRIBUTING.md

```diff
@@ -16,7 +18,48 @@ First off all, thank you for taking the time to contribute!
 
 Help us keep exercism welcoming. Please read and abide by the [Code of Conduct][coc].
 
-## Setting up your local development environment
+## What is the walkthrough?
+
+### Background
+
+Looking at the data, we noticed that a significant number of users have never
+completed a single exercise on exercism. We believe this is because setting it up
+isn't easy. This walkthrough aims to solve that.
+
+To do exercises on exercism, installing a CLI is required. This walkthrough aims
+to guide users throughout the several installation steps.
+
+### Structure
+
+The walkthrough is strucured in the following way:
+
+1. The walkthrough is a step-by-step guide. This should prevent users from
+getting lost.
+1. At the end of each step, the user must be able to confirm whether they've
+done the step correctly. This should prevent users from installing incomplete
+packages which may lead to harder troubleshooting in the future.
+1. When the user encounters a problem during a step, the user should be given
+steps to self-diagnose or troubleshoot the problem on their own.
+
+## How can I help?
+
+Again, thank you for taking the time to contribute! You could contribute to
+the walkthrough by doing these:
+
+1. Giving feedback on the walkthrough. Answering questions such as,
+**How can we improve it?**, **What step was the most confusing?**,
+**What additional steps did you have to take?** would help us make the
+walkthrough cater to more people.
+1. Looking through the issues and submitting a patch. Upcoming features and
+bugs to fix would all be in this project's issue tracker.
+1. Testing out the walkthrough. As each user may have a unique setup, it'll
+be helpful to have a hand in testing out the walkthrough on different machines.
```

Comment:
I think we need to tweak this a little bit specifically to compare/contrast better with the first bullet point.

The first bullet point is about testing that the instructions are good, whereas I think this bullet point is about testing that the visual layout? Or is it about testing on different operating systems?

Reply:
Reading it again, I think the first bullet point means the same as this. I'll delete it.